apps/exports/runner.py

```diff
@@ -49,23 +56,43 @@ def _run_export_for_project(export_config, project, export_record, force):
     export_record.save()
     try:
         # pipe both stdout and stderr to the same place https://stackoverflow.com/a/41172862/8207
-        result = subprocess.run(
+        process = subprocess.Popen(
             _compile_export_command(export_config, project, force),
             stdout=subprocess.PIPE,
             stderr=subprocess.STDOUT,
             universal_newlines=True,
         )
+        log_buffer = _stream_log(process, export_record)
+        result = process.poll()
     except Exception as e:
         export_record.status = ExportRun.FAILED
         export_record.log = str(e)
     else:
-        export_record.status = _process_status_to_status_field(result.returncode)
-        export_record.log = result.stdout
+        export_record.status = _process_status_to_status_field(result)
+        export_record.log = "\n".join(log_buffer)
     export_record.completed_at = timezone.now()
     export_record.save()
     return export_record
 
 
+def _stream_log(process, export_record):
+    log_buffer = []
+    start_time = datetime.utcnow()
+    while True:
+        output = process.stdout.readline()
+        if process.poll() is not None and output == '':
+            break
+        if output:
+            log_buffer.append(output.strip())
```

Comment:
do we ever not want to strip this? e.g. an indented stack trace or something?

Reply:
I hadn't thought about that. I think we can just remove the `strip`
https://github.com/dimagi/commcare-sync/pull/41/commits/5b727328368bd31e2f7ec122532cbdc792dbea8f